code/game/objects/items/breakable.dm

```diff
@@ -0,0 +1,263 @@
+/////////////////////
+// Breakable Items //
+//  by Hinaichigo  //
+/////////////////////
+
+/obj/item
+	//Breakability:
+	var/breakable_flags /*Possible flags include BREAKABLE_ALL, BREAKABLE_HIT, BREAKABLE_UNARMED, BREAKABLE_WEAPON, BREAKABLE_AS_ALL, BREAKABLE_AS_THROWN, BREAKABLE_AS_MELEE, and BREAKABLE_NOMOB.
```

Comment:
I would just have the comment point to the file that contains the macros. No one is going to keep this comment updated.

Reply:
Done.